mslib/utils/get_content.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""
+
+    mslib.utils.get_content
+    ~~~~~~~~~~~~~~~~~~~~~~~
+
+    Returns the content of a markdown file as html
+
+    This file is part of MSS.
+
+    :copyright: Copyright 2008-2014 Deutsches Zentrum fuer Luft- und Raumfahrt e.V.
+    :copyright: Copyright 2011-2014 Marc Rautenhaus (mr)
+    :copyright: Copyright 2016-2023 by the MSS team, see AUTHORS.
```

Comment:
Are these copyright attributions correct (for this specific file)?

When I blame the get_content function on develop I get this:
```
$ git blame -C -C -C mslib/index.py
[...]
02bc9591d mslib/index.py (ReimarBauer   2023-08-21 21:43:09 +0200 129)     def get_content(filename, md_overrides=None, html_overrides=None):
d772b623f mslib/index.py (May           2021-07-01 13:37:56 +0200 130)         markdown = Markdown(extensions=["fenced_code"])
c1d0b364a mslib/index.py (Reimar Bauer  2020-09-06 21:50:30 +0200 131)         content = ""
c1d0b364a mslib/index.py (Reimar Bauer  2020-09-06 21:50:30 +0200 132)         if os.path.isfile(filename):
c1d0b364a mslib/index.py (Reimar Bauer  2020-09-06 21:50:30 +0200 133)             with codecs.open(filename, 'r', 'utf-8') as f:
eba98308e mslib/index.py (Reimar Bauer  2020-11-18 22:36:03 +0100 134)                 md_data = f.read()
eba98308e mslib/index.py (Reimar Bauer  2020-11-18 22:36:03 +0100 135)             md_data = md_data.replace(':ref:', '')
02bc9591d mslib/index.py (ReimarBauer   2023-08-21 21:43:09 +0200 136)             if md_overrides is not None:
02bc9591d mslib/index.py (ReimarBauer   2023-08-21 21:43:09 +0200 137)                 v1, v2 = md_overrides
09b1fd1e6 mslib/index.py (ReimarBauer   2021-04-09 13:22:04 +0200 138)                 md_data = md_data.replace(v1, v2)
eba98308e mslib/index.py (Reimar Bauer  2020-11-18 22:36:03 +0100 139)             content = markdown.convert(md_data)
02bc9591d mslib/index.py (ReimarBauer   2023-08-21 21:43:09 +0200 140)             if html_overrides is not None:
02bc9591d mslib/index.py (ReimarBauer   2023-08-21 21:43:09 +0200 141)                 v1, v2 = html_overrides
02bc9591d mslib/index.py (ReimarBauer   2023-08-21 21:43:09 +0200 142)                 content = content.replace(v1, v2)
c1d0b364a mslib/index.py (Reimar Bauer  2020-09-06 21:50:30 +0200 143)         return content
[...]
```

Shouldn't a copyright attribution for the MSS team be enough then?